src/components/feature_component.tsx

```diff
@@ -39,7 +40,7 @@ export const FeatureComponent = observer(function FeatureComponent({ feature, sh
 					value={feature.chosen}
 					onValueChanged={action(async () => {
 						await featureStore.toggleChosenFor(feature);
-						if (feature.type === 'unigram' && feature.chosen) domainStore.updateNgramFeatures();
+						if (feature.type === kFeatureTypeUnigram && feature.chosen) domainStore.updateNgramFeatures();
```

Comment:
It might better to use an enum (or an enum like object) here. But unfortunately enums are kind of a can of worms and I don't think we have a consistent approach for using them. I don't think you should change anything, I just thought of it while looking at the code.